HSLdevcom/jore-map-ui
# 1108 map objects selection with popup

Closes #1108 

Postgis PR: https://gitlab.hsl.fi/jore/jore-map-postgis/merge_requests/28

Popup to show links & nodes around click
* Implemented with POSTGIS func call (couldn't find a way to do this completely in frontend)
* Changed a bit how popups are rendered. Now you can pass data=any & type="popupType" to "openPopup" func that renders a popup component based on given popupType and passes data for that popup component. Popups can still be created with passing reactNode. Maybe there should be only one way to render a popup.

To fix hardcoded bufferSize: https://github.com/HSLdevcom/jore-map-ui/issues/1177

TO TEST:

pull postgis PR, run ./updatedb.sh (NOTIFY: this should now never get stuck! I fixed a delete query there that took like 15 - 45 min or sometimes was stuck forever)
